pages/api/add-additional-investors.js

```diff
@@ -0,0 +1,52 @@
+import { withIronSessionApiRoute } from "iron-session/next";
+
+import { db, admin } from "../../utils/firebase";
+//request needs profile.id
```

Comment:
Might be good to extract out some decorators like `@authed` in the future to handle this/extract out shared logic between backend routes